rfcs/text/0000-deprecate-log-original.md

```diff
@@ -0,0 +1,106 @@
+# 0000: Name of RFC
```

Comment:
Let's update the template placeholder.

For the title, I suggest using the term `remove` over `deprecate` since we're proposing removing `log.original` entirely at the major in favor of keeping `event.original`.

In the proposal body, I do think deprecation is worth capturing as well here, though. For ECS, deprecation can be as simple as adding a notice in the field's description ([`http.request.method`](https://www.elastic.co/guide/en/ecs/current/ecs-http.html#field-http-request-method) original case guidance, for example) discouraging the use of field since it'll be removed in the next major.

Reply:
applied